### PR TITLE
chore: refactor visualization-boxes.test.ts to run with more reliability

### PR DESCRIPTION
**Pull request by Shobhit1**
#### Description of changes

This PR is a result of experiments which played around removing a test case (`Headings`) from our `visualization-boxes.test.ts`. I experimented with various order to run the test and removing `headings` and placing `Automated Checks` last in the `test.each` order gave the best results.

The final solution to this seems like giving each test its own browser instance inturn removing the dependency of our test to work only if previous visualizations were removed; (that was the highest point of our test failures).
 

#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Added relevant unit test for your changes. (`yarn test`)
- [x] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] Ran precheckin (`yarn precheckin`)
- [ ] (UI changes only) Added screenshots/GIFs to description above
- [ ] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

File: src/tests/end-to-end/tests/target-page/visualization-boxes.test.ts [hunk 1]
```diff
@@ -12,26 +12,21 @@ describe('Target Page visualization boxes', () => {
...
         browser = await launchBrowser({ suppressFirstTimeDialog: true });
         targetPage = await browser.newTargetPage();
         popupPage = await browser.newPopupPage(targetPage);
         await popupPage.gotoAdhocPanel();
     });
 
-    beforeEach(async () => {
-        await popupPage.disableAllToggles();
-        await targetPage.waitForVisualizationBoxesToDisappear();
-    });
```
Comment (lisli1) [inline src/tests/end-to-end/tests/target-page/visualization-boxes.test.ts]: Are `disableAllToggles()` and `waitForVisualizationBoxesToDisappear()` used elsewhere? If not, should we remove the unused code?

**Comment by smoralesd** [comment]
Since each test will have it's own browser instance, did you measure execution time? how much did we increase build time?

File: src/tests/end-to-end/tests/target-page/visualization-boxes.test.ts (hunk 1 above)
Comment (Shobhit1) [reply src/tests/end-to-end/tests/target-page/visualization-boxes.test.ts]: I will check and update this.

**Comment by Shobhit1** [comment]
> Since each test will have it's own browser instance, did you measure execution time? how much did we increase build time?

There is a ~17 second increase in the overall time for the `e2e-reliability` build, which as for me, is not much if it gives a reliable output.

File: src/tests/end-to-end/tests/target-page/visualization-boxes.test.ts (hunk 1 above)
Comment (Shobhit1) [reply src/tests/end-to-end/tests/target-page/visualization-boxes.test.ts]: Done. Thanks for catching that.

File: src/tests/end-to-end/common/page-controllers/target-page.ts [hunk 2]
```diff
@@ -39,12 +39,4 @@ export class TargetPage extends Page {
         const shadowRoot = await this.getShadowRoot();
         return await formatChildElementForSnapshot(shadowRoot, '#insights-shadow-container');
     }
-
-    public async waitForVisualizationBoxesToDisappear(): Promise<void> {
-        const insightsRootContainer = await this.getSelectorElement(TargetPageInjectedComponentSelectors.insightsRootContainer);
-        if (insightsRootContainer) {
-            const shadowRoot = await this.getShadowRoot();
-            await this.waitForDescendentSelectorToDisappear(shadowRoot, TargetPageInjectedComponentSelectors.insightsVisualizationBox);
```
Comment (lisli1) [inline src/tests/end-to-end/common/page-controllers/target-page.ts]: It seems like `waitForDescendentSelectorToDisappear` is also only used here

Comment (Shobhit1) [reply src/tests/end-to-end/common/page-controllers/target-page.ts]: Done.

Comment (Shobhit1) [reply src/tests/end-to-end/common/page-controllers/target-page.ts]: Removed this function too as its not used anymore!

**Pull request merged**